### PR TITLE
games-emulation/{mednafen,mgba,sameboy}-jg: version bumps

### PR DESCRIPTION
**Pull request by orbea**

### CODE REVIEW

File: games-emulation/sameboy-jg/sameboy-jg-9999.ebuild [hunk 1]
```diff
@@ -15,7 +15,7 @@ if [[ "${PV}" == *9999 ]] ; then
 else
 	SRC_URI="https://gitlab.com/jgemu/${MY_PN}/-/archive/${PV}/${MY_P}.tar.bz2"
 	S="${WORKDIR}/${MY_P}"
-	KEYWORDS="~amd64"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86"
```
Comment (ConiKost) [inline games-emulation/sameboy-jg/sameboy-jg-9999.ebuild:18]: You are adding lot's of arches. Did you test on all of them?

Comment (orbea) [reply games-emulation/sameboy-jg/sameboy-jg-9999.ebuild:18]: Yes, see issue https://bugs.gentoo.org/891201. Here I am only updating the live ebuilds accordingly.

**Comment by orbea** [comment]
@thesamesam I would really appreciate if you reviewed this.

**Comment by thesamesam** [comment]
yes I'm already going through many PRs and will be doing more later

**Comment by orbea** [comment]
I updated sameboy-jg in this PR from `0.16.1` to the newly released `0.16.2`, there aren't really any major changes.

**Pull request closed without merging**